src/devices/Tcs3472x/InterruptState.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Tcs3472x
+{
+    public enum InterruptState
+    {
+        // Every RGBC cycle generates an interrupt
+        All = 0b0000,
+        // 1 clear channel value outside of threshold range
```

Comment:
I think one larger comment explaining what this means would be useful (no need for similar comment per value)